src/hearings/containers/request-hearing/hearing-edit-summary/hearing-edit-summary.component.ts

```diff
@@ -459,7 +459,15 @@ export class HearingEditSummaryComponent extends RequestHearingPageFlow implemen
     const interpreterLanguagesSHV = this.extractInterpreterLanguages(SHVIndividualParties);
     const interpreterLanguagesHMC = this.extractInterpreterLanguages(HRMIndividualParties);
 
-    return !_.isEqual(interpreterLanguagesSHV, interpreterLanguagesHMC);
+    // return !_.isEqual(interpreterLanguagesSHV, interpreterLanguagesHMC);
```

Comment:
Does this comment still need to be in?

Reply:
comment left in in error, this has been removed now. Thank you